docs/team/sharpstorm.md

```diff
@@ -5,50 +5,64 @@ title: Zong Han's Project Portfolio Page
+  * Test cases for the Appointments subsystem, as well as overall test case maintenance of the `model` package.
 
 * **Project management**:
-  * Served as the overall project lead in organizing meetings and working with the entire team to ensure that integration of components proceed smoothly.
-  * Served as the project schedule manager in reminding team members of deadlines, and setting internal deadlines to ensure that the project milestone deadline can be met.
+  * Served as the **overall project lead** in organizing meetings and working with the entire team to ensure that integration of components proceed smoothly.
+  * Served as the **project schedule manager** to manage internal deadlines across all team members to ensure that the project milestone deadlines can be met.
+  * Upkeep of code quality and coverage across all members
 
-* **Enhancements to existing features**:
-  * Added UI result display mode switching
+* **Team Based Tasks**
+  * Upkeep shared team components in the Developer and User Guides, including User Stories and Product Scope
+  * Assisted in the management of the team's issue tracker
 
 * **Documentation**:
   * User Guide:
     * Added documentation for all appointment related functionalities, including:
-      * `addappointment`
-      * `listappointments`
-      * `editappointment`
-      * `deleteappointment`
-      * `appointmentsbetween`
-    * Section on common date and time formats accepted by the application
+      * `addappt`, `editappt`, `deleteappt`, `listappt`, `apptbetween`, `freebetween`
+    * Improved `addperson` and `editperson` to include parameter constraints
+    * Added section for [Common Date and Time Formats](https://ay2122s2-cs2103-w17-1.github.io/tp/UserGuide.html#common-date-and-time-syntax) accepted by the application
+    * Added section for [Global Application Input Constraints](https://ay2122s2-cs2103-w17-1.github.io/tp/UserGuide.html#global-input-constraints)
   * Developer Guide:
-    * Added section for appointment models
-    * Added section for the appointments filtering feature
-    * Added section for schedule inflation and serialization
-    * Added section for UI result display mode switching
+    * Added implementation and architectural design details of the `Appointment` models
+    * Added implementation details of the `Appointment` filtering feature
+    * Added implementation details for `Schedule` inflation and serialization
+    * Added implementation and architectural design details for Date Time Input Parsing
+    * Added implementation details for UI result display mode switching
+    * Added use cases for Appointments
 
 * **Community**:
-  * `to be added soon`
-
-* **Tools**:
-  * `to be added soon`
+  * Reviewed a significant amount of PRs within the team ([347 comments](https://nus-cs2103-ay2122s2.github.io/dashboards/contents/tp-comments.html) across [60+ PRs](https://github.com/AY2122S2-CS2103-W17-1/tp/pulls?q=is%3Apr+is%3Aclosed))
```

Comment:
I suggest for the across PRs that part, can use this filter instead
https://github.com/AY2122S2-CS2103-W17-1/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3Asharpstorm

it'll show what PRs u reviewed in particular

Reply:
Thanks, adopted in commit [1f0c97c](https://github.com/AY2122S2-CS2103-W17-1/tp/pull/317/commits/1f0c97c0db655a29be08cbf4049fe7f7a58f0945).